samples/bluetooth/direction_finding_connectionless_tx/child_image/hci_rpmsg.conf

```diff
@@ -7,15 +7,26 @@
 # Reguired to enable BT_BUF_CMD_TX_SIZE for LE Set Extended Advertising Data command
 CONFIG_BT_EXT_ADV=y
 
+CONFIG_BT_PER_ADV_SYNC=y
+
 CONFIG_BT_CTLR=y
 CONFIG_BT_LL_SW_SPLIT=y
 
 CONFIG_BT_CTLR_ADV_EXT=y
 CONFIG_BT_CTLR_ADV_PERIODIC=y
+CONFIG_BT_CTLR_SYNC_PERIODIC=y
 
 # Enable Direction Finding Feature including AoA and AoD
 CONFIG_BT_CTLR_DF=y
 
-# Disable Direction Finding Rx mode
-CONFIG_BT_CTLR_DF_SCAN_CTE_RX=n
+# Required to correctly setup number of PDUs in periodic advertising train
```

Comment:
train or chain ? :)

Reply:
The term is used interchangeable when you get deep into periodic advertising implementation but I'll change it to `chain` to be consistent.